src/components/form/__tests__/Form-test.tsx

```diff
@@ -50,6 +50,14 @@ it('should submit and reset the form', async () => {
   expect(onReset).toHaveBeenCalled();
 });
 
+it('should not prevent default on submit if action is defined', async () => {
```

Comment:
The test is raising errors:
![image](https://github.com/user-attachments/assets/2c0275fa-eb02-4f0f-8a4d-f2d4d94ec8bf)

Let's either expect the error, but seems a bit weird since it's most probably a jest related error more than a real one...